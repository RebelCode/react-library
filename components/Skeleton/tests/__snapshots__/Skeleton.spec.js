import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Skeleton } from "../../index";

afterEach(cleanup);

describe("Skeleton", () => {
  const Comp = (
    <Skeleton aria-label="Skeleton">
      <h1>One</h1>
      <h1>Two</h1>
    </Skeleton>
  );

  it("should take a snapshot", () => {
    const { asFragment } = render(Comp);

    expect(asFragment(Comp)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(Comp);

    expect(queryByLabelText("Skeleton")).toBeTruthy();
  });
});
