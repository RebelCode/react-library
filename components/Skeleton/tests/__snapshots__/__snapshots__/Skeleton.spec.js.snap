// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should take a snapshot 1`] = `
<DocumentFragment>
  <div
    aria-label="Skeleton"
    class="loading"
  >
    <h1>
      One
    </h1>
    <h1>
      Two
    </h1>
  </div>
</DocumentFragment>
`;
